rocAL_pybind/amd/rocal/types.py

```diff
@@ -147,6 +150,7 @@
     MAX_SIZE_ORIG: ("MAX_SIZE_ORIG", MAX_SIZE_ORIG),
     USER_GIVEN_SIZE_ORIG: ("USER_GIVEN_SIZE_ORIG", USER_GIVEN_SIZE_ORIG),
 
+    NONE: ("NONE", NONE),
```

Comment:
Is it used?

Reply:
Used as the default layout in numpy reader function if user doesn't specify a layout

Reply:
Do we have equivalent type in C++?